lib/ask/file_info.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Ask.FileInfo do
+  @doc """
+    Retrieves informations about one or multiple files, using the `file` command
+    line tool which is shipped with most linux distributions.
+  """
+  @spec get_info(Path.t() | [Path.t()]) :: %{Path.t() => String.t()}
+  def get_info(names)
+  def get_info(name) when is_binary(name), do: get_info([name])
+
+  def get_info(names) when is_list(names) do
+    {result, 0} = System.cmd("file", ["--mime-type" | names])
```

Comment:
I wouldn't add a dependency on yet another external tool (and parsing its output) if we are any way going to ask Sox to detect the type and convert it itself.

Reply:
I noticed afterwards that we use `soxi` in specs, but it does exactly the same (relies on `libmagic`, and the `file` tool is more ubiquitous (I checked it on ubuntu/gnu and alpine/musl) and the output was identical.